examples/deploy/terraform/infra/variables.tf

```diff
@@ -153,6 +247,7 @@ variable "kms" {
 
 variable "eks" {
   description = <<EOF
+    creation_role_name = Only meant to support an imported role.
```

Comment:
This isn't quite clear to me. Maybe we should be more descriptive, like `Name of the role to import` or `Role with adequate migration permissions`?